assignments/week_1/exercise_1_1/exercise_1_1.md

```diff
@@ -0,0 +1,161 @@
+# Exercise 1.1
+Welcome to your first exercise of RoboJackets software training! Exercises are shorter and simpler assignments where you will be walked through how to write certain code. This exercise will be testing your knowledge of basic C++ syntax as well as basic programming concepts. For this exercise we expect you to have watched or know the content of the Codecademy C++ course up through vectors.
+
+# Exercise Objective
```

Comment:
Can we include a picture of an example robot with a graphic. I am having trouble getting an idea of what the robot configuration is.